addons/website/views/snippets/s_rating.xml

```diff
@@ -2,41 +2,60 @@
 <odoo>
 
 <template id="s_rating" name="Rating">
-    <div class="s_rating row">
-        <div class="col-lg-12 s_rating_stars s_rating_3 s_rating_1x pt16 pb16">
-            <h4>Quality</h4>
-            <i class="fa fa-1x"/>
-            <i class="fa fa-1x"/>
-            <i class="fa fa-1x"/>
-            <i class="fa fa-1x"/>
-            <i class="fa fa-1x"/>
-            <div class="s_rating_bar"/>
+    <div data-vcss="001" data-icon="fa-star" data-active-custom-icon="" data-inactive-custom-icon="" class="s_rating pt16 pb16">
+        <h4 class="s_rating_title">Quality</h4>
+        <div class="s_rating_icons o_not_editable">
+            <span class="s_rating_active_icons">
+                <i class="fa fa-fw fa-star"/>
+                <i class="fa fa-fw fa-star"/>
+                <i class="fa fa-fw fa-star"/>
+            </span>
+            <span class="s_rating_inactive_icons">
+                <i class="fa fa-fw fa-star-o"/>
+                <i class="fa fa-fw fa-star-o"/>
+            </span>
         </div>
     </div>
 </template>
 
 <template id="s_rating_options" inherit_id="website.snippet_options">
     <xpath expr="." position="inside">
-        <div data-selector=".s_rating > div">
+        <div data-js="Rating" data-selector=".s_rating">
             <we-select string="Icon">
-                <we-button data-select-class="s_rating_stars"><i class="fa fa-fw fa-star"/> Stars</we-button>
-                <we-button data-select-class="s_rating_squares"><i class="fa fa-fw fa-square"/> Squares</we-button>
-                <we-button data-select-class="s_rating_hearts"><i class="fa fa-fw fa-heart"/> Hearts</we-button>
-                <we-button data-select-class="s_rating_bar"><i class="fa fa-fw fa-tasks"/> Progress Bar</we-button>
-            </we-select>
-            <we-select string="Value">
-                <we-button data-select-class="">0</we-button>
-                <we-button data-select-class="s_rating_1">1/5</we-button>
-                <we-button data-select-class="s_rating_2">2/5</we-button>
-                <we-button data-select-class="s_rating_3">3/5</we-button>
-                <we-button data-select-class="s_rating_4">4/5</we-button>
-                <we-button data-select-class="s_rating_5">5/5</we-button>
+                <we-button data-set-icons="fa-star"><i class="fa fa-fw fa-star"/> Stars</we-button>
+                <we-button data-set-icons="fa-thumbs-up"><i class="fa fa-fw fa-thumbs-up"/> Thumbs</we-button>
+                <we-button data-set-icons="fa-circle"><i class="fa fa-fw fa-circle"/> Circles</we-button>
+                <we-button data-set-icons="fa-square"><i class="fa fa-fw fa-square"/> Squares</we-button>
+                <we-button data-set-icons="fa-heart"><i class="fa fa-fw fa-heart"/> Hearts</we-button>
+                <we-button data-set-icons="fa-custom" class="d-none">Custom</we-button>
```

Comment:
You can remove the d-none class, it will normally be automatically added. Better to not add it here.

Reply:
We need it cause we dont want to show 'custom' in the select except if its selected.. or maybe I miss something ? ;)

Reply:
> it will normally be automatically added

? Or I am missing something ?